__tests__/api.test.js

```diff
@@ -169,6 +169,34 @@ describe('GET/api/articles(sort queries)',()=>{
       });
   });
 })
+describe('GET/api/articles(topic query)',()=>{
+  it("status 200: responds with articles of given topic", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({ body }) => {
+        body.articles.forEach((item) => {
+          expect(item.topic).toBe("cats");
+        });
+      });
+  });
+  it("status 400: returns an error message when an invalid value is given", () => {
+    return request(app)
+      .get("/api/articles?topic=apple")
+      .expect(400)
+      .then(({body}) => {
+        expect(body.msg).toBe("Invalid Topic");
+      });
+  });
+  it("status 400: returns an error message when given value is empty", () => {
+    return request(app)
+      .get("/api/articles?topic=")
+      .expect(400)
+      .then(({body}) => {
+        expect(body.msg).toBe("Invalid Topic");
+      });
```

Comment:
If the topic exists but has no articles yet then we should return a 200 and an empty array. If you think about how you could use this in the front end, an empty array could be displayed on the screen in the same way an array of articles could be displayed, just with a message saying there's no articles yet rather than an error. 

The topic does exist, so it shouldn't give an error